screens/SettingsScreen.js

```diff
@@ -34,201 +225,53 @@ class SettingsScreen extends React.Component {
           <Pages indicatorColor={mainThemeColor}>
             <View style={{flex: 1, paddingBottom: 16}}>
               <View style={[styles.menuContainer]}>
-                <MenuButton
-                  route='Account'
-                  onPress={() => this.props.navigation.navigate('Account')}
-                  title={t('settings.account')}
-                  icon={
-                    <MaterialIcon
-                      name="account-box"
-                      size={40}
-                      style={[styles.buttonIconStyle]}
-                    />
-                  } />
+                {menuButtonsArr?.[0] ?? <View style={{flex: 1}}></View>}
```

Comment:
I think this implementation can be refactored to use a loop instead of hard coded index right?